extra_data/tests/conftest.py

```diff
@@ -8,83 +8,84 @@
 from . import make_examples
 
 
+@pytest.fixture(scope='session', params=['0.5', '1.0'])
+def format_version(request):
+    return request.param
```

Comment:
I'm not totally keen on parameterising every mock data test like this. pytest makes it really easy to do this, and it's good if you do need to test how different options interact, but for most of what we're testing the changes in format 1.0 don't really matter, so running the test twice is a bit of a waste. This currently adds about 30 seconds to running the tests, and that will increase as we keep adding tests.

We don't necessarily need to address this now - 30 seconds extra isn't *that* bad - I just want to register that at some point we'll want to be more selective.

Reply:
Yep, I thought about this as well. Probably only a subset of the tests would require testing on different version but as you said, this has not a big impact at the moment.
We could also think different tests strategies, simple/fast on branches and extended tests when merging to master, or in a nightly build, ... (we could then also run much longer tests  download real data, processing, ...)

Reply:
We could, but different test strategies and ways of switching between them means more complexity, so I'd rather put it off if possible.

Maybe the biggest win in that direction would be if we had a quick automated way of finding a newly written data file on Maxwell, we could try to have a nightly test using a new file, to alert us if something unexpected happened to the format. Though we're getting something a bit like that by running validation in the calibration pipeline.